src/main/java/andelu/Storage.java

```diff
@@ -0,0 +1,211 @@
+package andelu;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+
+
+/**
+ * A Storage class that acts as a File Manager to handle all file's read and write method.
+ */
+public class Storage {
+
+    /** A fixed directory for text files. */
+    static final Path DIRECTORY_PATH = Paths.get("./data");
+
+    /** The name of the file. */
+    private String fileName;
+
+    /** The path for the file. */
+    private Path filePath;
+
+
+    /**
+     * A constructor to create new Storage Object.
+     *
+     * @param fileName The file name to be loaded or stored.
+     */
+    public Storage(String fileName) {
+        this.fileName = fileName;
+        filePath = DIRECTORY_PATH.resolve(fileName + ".txt");
+        createFile();
+    }
+
+    /**
+     * Creates directory and file if it doesn't exist.
+     */
+    private void createFile() {
+        try {
+            if (!Files.exists(DIRECTORY_PATH)) {
+                Files.createDirectories(DIRECTORY_PATH);
+            }
```

Comment:
### #General
Good job identifying the need to create the directory when it is absent. I have missed this in my code.